src/utils/client.ts

```diff
@@ -160,57 +168,87 @@ export async function getEvent({serverId, eventId}: EventLocator, options?: obje
   return makeRequest<IndicoEvent>(serverId, `api/checkin/event/${eventId}/`, options);
 }
 
-export async function getRegforms({serverId, eventId}: EventLocator, options?: object) {
-  return makeRequest<IndicoRegform[]>(serverId, `api/checkin/event/${eventId}/forms/`, options);
+export async function getRegforms(
+  {serverId, eventId}: EventLocator,
+  checkTypeId?: number,
+  options?: object
+) {
+  const url = checkTypeId
+    ? `api/checkin/event/${eventId}/forms/?check_type_id=${checkTypeId}`
+    : `api/checkin/event/${eventId}/forms/`;
+  return makeRequest<IndicoRegform[]>(serverId, url, options);
 }
 
-export async function getRegform({serverId, eventId, regformId}: RegformLocator, options?: object) {
-  return makeRequest<IndicoRegform>(
-    serverId,
-    `api/checkin/event/${eventId}/forms/${regformId}/`,
-    options
-  );
+export async function getRegform(
+  {serverId, eventId, regformId}: RegformLocator,
+  checkTypeId?: number,
+  options?: object
+) {
+  const url = checkTypeId
+    ? `api/checkin/event/${eventId}/forms/${regformId}/?check_type_id=${checkTypeId}`
+    : `api/checkin/event/${eventId}/forms/${regformId}/`;
+  return makeRequest<IndicoRegform>(serverId, url, options);
 }
 
 export async function getParticipants(
   {serverId, eventId, regformId}: RegformLocator,
+  checkTypeId?: number,
   options?: object
 ) {
-  return makeRequest<IndicoParticipantList>(
-    serverId,
-    `api/checkin/event/${eventId}/forms/${regformId}/registrations/`,
-    options
-  );
+  const url = checkTypeId
+    ? `api/checkin/event/${eventId}/forms/${regformId}/registrations/?check_type_id=${checkTypeId}`
+    : `api/checkin/event/${eventId}/forms/${regformId}/registrations/`;
```

Comment:
```suggestion
  const url = `api/checkin/event/${eventId}/forms/${regformId}/registrations/${checkTypeId ? `?check_type_id=${checkTypeId}` : ''}`;
```

You can also do the same for the rest of your `checkTypeId` URLs.